ast-Update: 2017-06-29
Forwarded: not-needed
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=764226
Bug-Upstream: http://www.midnight-commander.org/ticket/3285
Author: Dmitry Smirnov <onlyjob@member.fsf.org>
Updated-by: Denis Briand <debian@denis-briand.fr>
Description: 'mcedit' status line to show full path to file, like in 'mcview'.

--- a/doc/man/mcedit.1.in
+++ b/doc/man/mcedit.1.in
@@ -604,9 +604,9 @@
 will be untouched. Default value is
 "\fB-\fR\fB+\fR\fB*\fR\fB\\\fR\fB,\fR\fB.\fR\fB;\fR\fB:\fR\fB&\fR\fB>\fR".
 .TP
 .I editor_state_full_filename
-Show full path name in the status line. If disabled (default), only base name of the
+Show full path name in the status line. If disabled, only base name of the
 file is shown.
 .SH MISCELLANEOUS
 The editor also displays non\-us characters (160+).  When editing
 binary files, you should set
--- a/src/editor/edit.c
+++ b/src/editor/edit.c
@@ -87,9 +87,9 @@
     .persistent_selections = TRUE,
     .drop_selection_on_copy = TRUE,
     .cursor_beyond_eol = FALSE,
     .cursor_after_inserted_block = FALSE,
-    .state_full_filename = FALSE,
+    .state_full_filename = TRUE,
     .line_state = FALSE,
     .line_state_width = 0,
     .save_mode = EDIT_QUICK_SAVE,
     .confirm_save = TRUE,
